domains/transfer/TransferAgreementAutomation/Worker/Service/ProjectOriginWalletService.cs

```diff
@@ -117,20 +117,20 @@ private async Task<RepeatedField<GranularCertificate>> GetGranularCertificates(M
         return response.GranularCertificates;
     }
 
-    private static bool IsPeriodMatching(TransferAgreementDto transferAgreement, GranularCertificate certificate)
+    private static bool IsPeriodMatching(TransferAgreement transferAgreement, GranularCertificate certificate)
```

Comment:
Do we have a tight coupling with certificates subsystem now?

Reply:
not more than before